src/haxelib/api/GlobalScope.hx

```diff
@@ -21,6 +23,9 @@ using StringTools;
 class GlobalScope extends Scope {
 	function new(repository:Repository) {
 		super(false, repository);
+		
+		// While this class is "GlobalScope", there's currently not a "LocalScope" equivalent class. For now we can add this here. 
+		isLocal = repository.path != RepoManager.getGlobalPath();
```

Comment:
Hmm, `RepoManager.getGlobalPath()` will throw if there's no global repository setup.

This not so pretty hack might not be a good idea..
`isLocal` should probably live in `Repository` itself, which could also more easily identify local / global cases (might need to rework `get()` a bit there).